src/panels/config/entities/entity-aliases/dialog-entity-aliases.ts

```diff
@@ -59,6 +59,7 @@ class DialogEntityAliases extends LitElement {
           "ui.dialogs.entity_registry.editor.aliases.heading",
           { name }
         )}
+        .suppressDefaultPressSelector=${"input"}
```

Comment:
Should it not also have the defaults?

Reply:
It will disable the behavior for all prompt dialogs 😞

Reply:
No, I mean you now overrule `suppressDefaultPressSelector`, to only be `input`, all the default suppressed elements (like button) are not suppressed now for this dialog?

Reply:
No, dialog base merges it

Reply:
Why not just list all the elements that we want to suppress in ha-dialog (this is what mwc-dialog does)? Making it an input seems unnecessary.

Reply:
For now I stopped propagation from the textfield and added button to the list. 
I agree @steverep, we should have a list based on accessibility criteria. I will look to do this in another PR

Reply:
Maybe I'm missing what you're trying to do here, but I'm concerned you're breaking enter key behavior that will be universally expected for any keyboard user. The only elements that shouldn't close a dialogue on enter would be multi line fields or open drop downs.

I can't do it now, but later tonight or tomorrow, I can take a closer look and pull the branch down.

Reply:
The aliases are in a list of input field. Pressing enter add a new line (new input field). User can delete the line by clicking the delete button next to the input field. User can add a new input field by clicking "add alias" button.

I'll be interested by your feedback how we can provide a good accessibility support for list of input field.

Reply:
That part I get, and pressing enter to add the field to the list is fine, although some helper text to advise of that behavior would be nice.  I was more concerned with having `button` or `input` in the suppress list.  Let me check it out.

More generally though, I think the list of input fields used here (and in various card editor dialogs) is just a very clunky experience for keyboard users.  It takes too many strokes to accomplish something simple like just reading the list or finding and deleting an item.  I would much rather see a single text field to add/edit, and the items already in the list held as a listbox with text/icons or chips.